components/MaskedTransition/src/private/MDCMaskedTransitionMotionSpec.m

```diff
@@ -0,0 +1,175 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCMaskedTransitionMotionSpec.h"
+
+#define MDMEightyForty _MDMBezier(0.4f, 0.0f, 0.2f, 1.0f)
```

Comment:
We're not in the MDM namespace here, so MDC if any prefix.

Also, macros are nasty—can you name these something self-explanatory, or comment?